### PR TITLE
all: add test for package comments, fix, add comments as needed

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #cleanup

### CODE REVIEW

**Review by maisem — APPROVED** [review]

File: tstest/integration/vms/gen/test_codegen.go [hunk 1]
```diff
@@ -1,51 +0,0 @@
-// Copyright (c) Tailscale Inc & AUTHORS
```
Comment (maisem) [inline tstest/integration/vms/gen/test_codegen.go:1]: unrelated?

Comment (bradfitz) [reply tstest/integration/vms/gen/test_codegen.go:1]: I went to add a package doc to it and then found it was unused, as far as I could tell, so I fixed the package doc by deleting it.
